commands/audit/sca/python/python.go

```diff
@@ -103,9 +115,63 @@ func getDependencies(auditPython *AuditPython) (dependenciesGraph map[string][]s
 		sca.LogExecutableVersion("python")
 		sca.LogExecutableVersion(string(auditPython.Tool))
 	}
+	if auditPython.IsCurationCmd {
+		pipUrls, errProcessed := processPipDownloadsUrlsFromReportFile()
+		if errProcessed != nil {
+			err = errProcessed
+			return
+		}
+		params.SetDownloadUrls(pipUrls)
+	}
 	return
 }
 
+func processPipDownloadsUrlsFromReportFile() (map[string]string, error) {
+	exist, err := fileutils.IsFileExists("report.json", false)
+	if err != nil {
+		return nil, err
+	}
+	if !exist {
+		err = errors.New("process failed, report file wasn't found, cant processed with curation command")
+		return nil, err
+	}
+	var reportBytes []byte
+	reportBytes, err = fileutils.ReadFile("report.json")
+	if err != nil {
+		return nil, err
+	}
+	pipReport := &pypiReport{}
+	if err = json.Unmarshal(reportBytes, pipReport); err != nil {
+		return nil, err
+	}
```

Comment:
```suggestion
    const pythonReportFile = "report.json"
    func readPipReportFileIfExists() (pipReport *pypiReport, err error) {
        if exist, err := fileutils.IsFileExists(pythonReportFile, false); err != nil {
		    return
	    } else if !exist {
		    err = errors.New("Process failed: report file not found. Unable to proceed with curation command.")
		    return
	    }
        var reportBytes []byte
        if reportBytes, err = fileutils.ReadFile(pythonReportFile); err != nil {
		    return
	    }
        pipReport = &pypiReport{}
        err = json.Unmarshal(reportBytes, pipReport)
		return
	}
```
extract operation to a separated method for readability

Reply:
Fixed